src/GameServer.ts

```diff
@@ -0,0 +1,51 @@
+const socketIo = require('socket.io');
+const cors = require('cors');
+import * as express from 'express';
+import { createServer, Server } from 'http';
+import {GameEvent} from './Constants';
+
+// GameServer Properties
+export class GameServer {
+    public static readonly PORT: number = 8080;
```

Comment:
Typescript does quite a good job of inferring type definitions, so in cases like this it knows that `PORT` will be a number since your assigning a number to it, so dont feel like you have to explicitly declare it.